src/main/java/de/qabel/desktop/DesktopClientGui.java

```diff
@@ -55,63 +54,67 @@ public void start(Stage stage) throws Exception {
-                    logger.error("failed to init background services: " + e.getMessage(), e);
-                    Platform.runLater(() -> {
-                        final CrashReportAlert alert = new CrashReportAlert(
-                            services.getCrashReportHandler(),
-                            "failed to init brackground services",
-                            e
-                        );
-                        alert.showAndWait();
-                        System.exit(-1);
-                    });
-                }
-            })
+            initBackgroundServices()
         );
         config.onSelectIdentity(this::addShareMessageRenderer);
+        primaryStage.show();
 
-        QabelTray tray = new QabelTray(primaryStage, new AwtToast());
-        tray.install();
-        Runtime.getRuntime().addShutdownHook(new Thread() {
-            @Override
-            public void run() {
-                Platform.exit();
+        QabelTray tray = getQabelTray();
+        trayNotifications(tray);
+
+        if (config.hasAccount()) {
+            primaryStage.close();
+            System.out.println(" primaryStage closed ");
+        }
+    }
+
+    @NotNull
+    private Future<?> initBackgroundServices() {
+        return executorService.submit(() -> {
+            try {
+                startTransferManager();
+                startSyncDaemon();
+                startDropDaemon();
+
+                loggedInStage();
+                if (config.getSelectedIdentity() != null) {
+                    addShareMessageRenderer(config.getSelectedIdentity());
+                }
+            } catch (Exception e) {
+                logger.error("failed to init background services: " + e.getMessage(), e);
+                Platform.runLater(() -> {
+                    final CrashReportAlert alert = new CrashReportAlert(
+                        services.getCrashReportHandler(),
+                        "failed to init brackground services",
+                        e
+                    );
+                    alert.showAndWait();
+                    System.exit(-1);
+                });
             }
         });
-        primaryStage.show();
+    }
+
+    private void loggedInStage() {
+        view = new LayoutView();
+        Parent view = this.view.getView();
+        runtimeConfiguration.setWindow(((LayoutController) this.view.getPresenter()).getWindow());
+        Scene layoutScene = new Scene(view, 900, 600, true, SceneAntialiasing.BALANCED);
+        Platform.runLater(() -> primaryStage.setScene(layoutScene));
+    }
+
+    private void doLoginStage() {
+        Scene scene;
+        primaryStage.getIcons().setAll(new Image(getClass().getResourceAsStream("/logo-invert_small.png")));
+        Platform.setImplicitExit(false);
+        primaryStage.setTitle(getResources().getString("title"));
+        scene = new Scene(new LoginView().getView(), 370, 570, true, SceneAntialiasing.BALANCED);
+        primaryStage.setScene(scene);
```

Comment:
Only the last two lines are related to the LoginStage. Either revert/inline it or rename the method ... setUpWindow? don't know...